xbmc/application/ApplicationVolumeHandling.h

```diff
@@ -23,10 +25,11 @@ class CApplicationVolumeHandling
   // times per second (saves doing settings lookup)
   struct ReplayGainSettings
   {
-    int iPreAmp;
-    int iNoGainPreAmp;
-    int iType;
-    bool bAvoidClipping;
+    float m_preAmp;
+    float m_noGainPreAmp;
+    ReplayGain::Type m_type;
+    bool m_avoidClipping;
+    ReplayGainSettings() : m_preAmp(0.0f), m_noGainPreAmp(0.0f), m_type(ReplayGain::NONE), m_avoidClipping(false) {}
```

Comment:
you need a constructor?

Reply:
@ksooo  [asked me to add one](https://github.com/xbmc/xbmc/pull/21881#discussion_r980956502) to be aligned with the current coding standard.

Reply:
initialize members not means add a constructor but initialize each single memeber variable

Reply:
Fixed